cylc/flow/task_events_mgr.py

```diff
@@ -108,6 +109,44 @@ def log_task_job_activity(ctx, suite, point, name, submit_num=None):
         LOG.debug(ctx_str)
 
 
+class EventData(Enum):
+    """Template variables which are available to event handlers."""
+
+    Event = 'event'
+    Suite = 'suite'
+    SuiteUUID = 'suite_uuid'
+    CyclePoint = 'point'
+    SubmitNum = 'submit_num'
+    TryNum = 'try_num'
+    ID = 'id'
+    Message = 'message'
+    BatchSysName = 'batch_sys_name'
+    BatchSysJobID = 'batch_sys_job_id'
+    SubmitTime = 'submit_time'
+    StartTime = 'start_time'
+    FinishTime = 'finish_time'
+    PlatformName = 'platform_name'
+    TaskName = 'name'
+    TaskURL = 'task_url'  # deprecated
+    SuiteURL = 'suite_url'  # deprecated
```

Comment:
BTW Python docs say:
> Because Enums are used to represent constants we recommend using UPPER_CASE names for enum members, and will be using that style in our examples.

Reply:
Yes, this was a purposful violation that slides through flake8 nicely, can change if it's unpopular.

Python enums are really strange in that what you get out is not what you get in:

```python
>>> from enum import Enum
>>> class Foo(Enum):
...     ANSWER = 42
... 
>>> Foo.ANSWER
<Foo.ANSWER: 42>
# wait that's not an integer

>>> Foo.ANSWER.value
42
# hmmm
```